dropshot/src/error.rs

```diff
@@ -86,7 +111,7 @@ pub struct HttpError {
     // by some enum type?
```

Comment:
I'm not sure about the first part of this TODO, but the "should this be parametrized by some enum type" seems (effectively?) resolved by your change here. i.e. in light of custom error types I don't think we'd do that.

Reply:
Yeah. Personally, I think we can probably drop the whole comment, since IMO, the
point of requiring the error code field is to encourage user code to report more
structured errors, which...we have a different solutuon for.